Components/Account/Shared/EquipmentAvailability.razor

```diff
@@ -0,0 +1,135 @@
+﻿@inject ApplicationDbContext DbContext;
```

Comment:
`Pages/EquipmentAvailability.razor` has changed from prior commits.  Make sure that this moved file has the changes to the original file, notably the `@code{ }` section.